dev-python/spyder/spyder-2.3.8.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+#
+EAPI=5
```

Comment:
Search for EAPI=6 in [PMS](https://projects.gentoo.org/pms/6/pms.html). One important change is, that patching is now natively support by the PM and there is no need to use the eutils eclass anymore.